.gitignore

```diff
@@ -73,3 +73,6 @@ package-lock.json
 
 # Generated Lockfiles
 Cartfile.resolved
+
+# Yalc
+sample/yalc.lock
```

Comment:
Should we delete the `yalc.lock` from the sample project? As at the moment it's still tracked in git.

Reply:
yeah, makes sense, I will delete it